pmlc/dialect/pxa/transforms/stencil.cc

```diff
@@ -169,6 +193,66 @@ bool Stencil::ComputeStrideInfo() {
   return tensorsStrides.size() == kNumTensors;
 }
 
+// Collect the non constant indices used to index the memref at specific index.
+// Ignore indices that are constant for the ParallelOp.
+BlockArgumentSet Stencil::UsedIdxs(unsigned strideInfoIndex) {
+  assert(strideInfoIndex < kNumTensors);
+
+  BlockArgumentSet used_idxs;
```

Comment:
Prefer `camelBack` over `snake_case`

Reply:
Yup, missed this.  Fix in the next commit...